runtime/src/bank.rs

```diff
@@ -1436,10 +1436,22 @@ impl Bank {
     /// calculation and could shield other real accounts.
     pub fn verify_snapshot_bank(&self) -> bool {
         self.purge_zero_lamport_accounts();
```

Comment:
btw, this purge process is introduced by [here](https://github.com/solana-labs/solana/pull/7005#discussion_r348936940). I guess this is now intended behavior.

Reply:
Yes.. that kind of got in by accident. Anatoly merged the PR with it in the stack, but I think it's ok, we can make it work.